spec/resources.rb

```diff
@@ -32,7 +32,13 @@ def self.exit_status(key)
 
   def self.shell(args)
     cmd = [Resources.instance.idm] + args
-    RunLoop::Shell.run_shell_command(cmd, {log_cmd: true, timeout: 180})
+
+    timeout = 180
+    if RunLoop::Environment.ci?
+      timeout = 300
```

Comment:
This is really disappointing. :/

Reply:
Most often, it passes very quick. But sometimes it takes more time for Xcode 11 (may be `beta` instability)